src/scripts/bttvIntegration.js

```diff
@@ -1,12 +1,6 @@
 function replaceKeywordWithEmoteImageString(textContent, keyword, imageSource, id){ 
-    if (!textContent) { 
-        throw "Text content is not provided!" 
-    } else if (typeof textContent !== "string"){ 
-        throw "Text content is not a string!"
-    }
-    const otherArgumentsAreValid = areValidInputs(keyword, imageSource, id); 
-    if (!otherArgumentsAreValid) {return textContent }
-
+    const otherArgumentsAreValid = areValidInputs(textContent, keyword, imageSource, id); 
+    if (!otherArgumentsAreValid) {return null }
```

Comment:
At this point, since it seems you're testing that all parameters are valid, maybe we rename this variable to `allArgumentsAreStrings` or something like that?

Reply:
LOL!! As I was going through to do the JSDocs comments, I already changed this to basically exactly what you're talking about, pushing that now